app/core/migrations/0002_user_current_job_title_user_current_skills_and_more.py

```diff
@@ -0,0 +1,85 @@
+# Generated by Django 4.0.7 on 2022-09-20 23:06
+
+from django.db import migrations, models
+import phonenumber_field.modelfields
+import timezone_field.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='user',
+            name='current_job_title',
+            field=models.CharField(blank=True, max_length=255),
+        ),
+        migrations.AddField(
+            model_name='user',
+            name='current_skills',
+            field=models.CharField(blank=True, max_length=255),
+        ),
+        migrations.AddField(
+            model_name='user',
+            name='first_name',
+            field=models.CharField(blank=True, max_length=255),
+        ),
+        migrations.AddField(
+            model_name='user',
+            name='github_handle',
+            field=models.CharField(blank=True, max_length=255),
+        ),
+        migrations.AddField(
+            model_name='user',
+            name='gmail',
+            field=models.EmailField(blank=True, max_length=254),
+        ),
+        migrations.AddField(
+            model_name='user',
+            name='last_name',
+            field=models.CharField(blank=True, max_length=255),
+        ),
+        migrations.AddField(
+            model_name='user',
+            name='linkedin_account',
```

Comment:
This could use help text. I wasn't sure if this should be the username or url or ??

Reply:
Will add help text. I think it should be the username, like the other similar fields. But this one has an ambiguous name.